ipatests/test_integration/test_krbtpolicy.py

```diff
@@ -238,6 +238,30 @@ def test_krbtpolicy_jitter_otp(self, reset_to_default_policy):
                                  "--user-auth-type", "otp"])
         kinit_check_life(self.master, USER1)
 
+    def test_krbtpolicy_jitter_otp_maxlife(self, reset_to_default_policy):
+        """Test jitter lifetime with OTP"""
+        master = self.master
+        reset_to_default_policy(self.master, USER1)
+        otpuid, hotp = add_otptoken(master, USER1, otptype="hotp")
+        master.run_command(["ipa", "otptoken-show", otpuid])
+
+        master.run_command(['ipa', 'krbtpolicy-mod', USER1,
+                            '--otp-maxlife=172800'])
+        master.run_command(["ipa", "user-mod", USER1,
+                            "--user-auth-type", "otp"])
+        otpvalue = hotp.generate(0).decode("ascii")
+        # basically kinit_otp
+        tasks.kdestroy_all(master)
+        master.run_command(["kinit", "-n", "-c", "/tmp/armor"])
+        master.run_command(
+            ["kinit", "-T", "-l", "2d", "/tmp/armor", USER1],
+            stdin_text=f"{PASSWORD}{otpvalue}\n",
+        )
+
+        result = master.run_command('klist | grep krbtgt')
+        assert maxlife_within_policy(result.stdout_text, 172800,
```

Comment:
Is the assertion with `is True` really necessary here? `maxlife_within_policy` returns only `bool` values.

Reply:
Fair point. I kept the style of the rest of the assertions in the test class.

Reply:
Also fair point, but the other comparisons with True in `test_ipahealthcheck` could potentially filter out truthy values, since they compare `json`-extracted values.

Reply:
I'm not sure I follow. This isn't related to healthcheck. I can strip out the True on the next push but I want to give CI a chance to finish up.